### ISSUE TITLE
Something into Readme

### ISSUE THREAD

**Issue by hyysam** [leichtrhino/nsf-torch#1]
Hello，Could you write the detail in the Readme to show how to use these codes?

**Comment by leichtrhino** [comment]
Hello,
Thanks for request for improving the documentation.
I wrote the brief explanation and essential usage (and not self-contained) of this repo in README.md .

**Comment by hyysam** [comment]
Thank you for your readme document, I haved trained with your codes, but the results are not ideal[https://github.com/hyysam/sample_of_ndf](url)，Can you share some demos？

**Comment by leichtrhino** [comment]
Thanks for providing your original/generated waveforms.

The two loss functions I showed use single framing/DFT configurations (shows in Table 1 in the [paper](https://arxiv.org/abs/1810.11946) ).
According to Figure 4 in the paper, the MOS score of the generated waveforms are not high when the model that has been trained with single configuration.

Please consider using all three framing/DFT configurations in Table 1.

Thanks.
